### PR TITLE
WINTERMUTE: Add Informer Alavi Beta version

### PR DESCRIPTION
**Pull request by aviloria**
Add 'Informer Alavi' Beta version.
Note: Dates in description are Persian-calendar based, as found in the developers page

### CODE REVIEW

File: engines/wintermute/detection_tables.h [hunk 1]
```diff
@@ -239,7 +239,12 @@ static const WMEGameDescription gameDescriptions[] = {
 	WME_WINENTRY("actualdest", "",
 		WME_ENTRY1s("data.dcp", "6926f44b26f21ceb1d840eaab9aeb510", 9081740), Common::EN_ANY, ADGF_UNSTABLE, WME_1_8_11),
 
-	// Informer Alavi - Murder of Miss Rojan (Persian) (Try version)
+	// Informer Alavi - Murder of Miss Rojan (Persian) (Beta version -old version-, between 1387/12 to 1392/6)
+	// NOTE: This is a 2.5D game that is out of ScummVM scope
```
Comment (sev-) [inline engines/wintermute/detection_tables.h:243]: Speaking of this. Maybe it makes sense to add a GF_3D or something, so we could show a message to the user? And of course, for the documentation purposes.

Comment (aviloria) [reply engines/wintermute/detection_tables.h:243]: @sev- For sure... I hope I'll be able to implement it in the right way

**Comment by sev-** [comment]
Thanks!

**Comment by lolbot-iichan** [comment]
Thanks for adding the Beta version! I have tried looking for it, but failed.

**Comment by aviloria** [comment]
You're welcome!
As it was a free beta publicly released, I think it could be shared with no problems:
https://mega.nz/file/Hc0RHI5Q#TdNw1SjGoAhqnDYJi2MTSUDTh80D9B3lRscHfTYoLHs

If there is a problem with that, feel free to remove the link.

**Pull request merged**